application/controllers/LimereplacementfieldsController.php

```diff
@@ -212,8 +212,26 @@ private function getChildQuestions(array $questions)
                 $question = "[{$row['subquestion2']}] " . $question;
             }
 
+            $replacementCode = $row['title'];
```

Comment:
This whole block looks very similar to collectQuestionReplacements()

Reply:
I'll check soon

Reply:
This function would not fully deliver what we need. I could probably use it for parts of the placeholder code creation,
but it would produce another unneccessary db call.
We already have all previous questions data via createFieldMap().

I did update all the docblocks in this controller and factored the new functionality into a separate function for better maintainability